packages/jest-resolve/src/__tests__/resolve.test.ts

```diff
@@ -707,3 +707,51 @@ describe('Resolver.getModulePaths() -> nodeModulesPaths()', () => {
     expect(dirs_actual).toEqual(expect.arrayContaining(dirs_expected));
   });
 });
+
+describe('findGlobalPaths', () => {
+  const paths = findGlobalPaths();
+  expect(paths.length).toBeGreaterThanOrEqual(1);
+});
+
+describe('Resolver.getGlobalPaths()', () => {
+  const _path = path;
+  let moduleMap: IModuleMap;
+  beforeEach(() => {
+    moduleMap = ModuleMap.create('/');
+  });
+
+  it('return global paths with npm package', () => {
+    jest.doMock('path', () => _path.posix);
+    const resolver = new Resolver(moduleMap, {} as ResolverConfig);
+    const globalPaths = resolver.getGlobalPaths('jest');
+    expect(globalPaths.length).toBeGreaterThan(0);
```

Comment:
Probably we could test this more precisely with something like

```suggestion
    globalPaths.forEach(globalPath => expect(require.resolve.paths('jest')).toContain(globalPath))
```

Reply:
Thanks! I will update it